components/server/src/iam/iam-oidc-create-session-payload.ts

```diff
@@ -4,45 +4,65 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+import { ErrorCodes } from "@gitpod/gitpod-protocol/lib/messaging/error";
+import { ResponseError } from "vscode-ws-jsonrpc";
```

Comment:
TODO: not depend on library for response shape. 
IDEA: duplicate in `gitpod-protocol`
BLOCKING: no